facebook/__init__.py

```diff
@@ -163,9 +163,20 @@ def put_object(self, parent_object, connection_name, **data):
 
         """
         assert self.access_token, "Write operations require an access token"
+
+        """
```

Comment:
We should be using [block comments](https://www.python.org/dev/peps/pep-0008/#block-comments) (the comments that start with `#`) instead of docstrings here.

Also, I think this comment could be a bit simpler, e.g. "Save file object to dictionary for uploading (if provided)."